lib/components.js

```diff
@@ -363,7 +363,9 @@ class DatePicker extends Component {
       'minimumDate',
       'minuteInterval',
       'mode',
-      'timeZoneOffsetInMinutes'
+      'timeZoneOffsetInMinutes',
+      'background',
```

Comment:
Ah... I see, you are right. With `tcomb-form-native` there's an additional problem with respect to `tcomb-form`: the platform problem. Logically the schema of the `options` prop should be:

- ... list of all props shared by both platforms (such as `help`)
- `ios` (optional object): contains the configuration of all "official" props supported by iOS (such as `minuteInterval` or `minimumDate`)
- `android` (optional object): contains the configuration of all "official" props supported by Android
- `config` (optional object): configuration fields specific to the template at hand

That would be a clean implementation and avoid name collisions. Maybe it's over-engeneering tough: the official props are already namespaced upstream (RN put an `Android` suffix) so perhaps it doesn't worth it to do such a big refactoring. What do you think?

In any case, I'd put all the specific fields of the template in the `config` object.